Ops: relayer container throws errors

**Describe the bug**
When running the private optimism setup with the relayer container enabled, it crashes with an internal error.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to `/ops`
2. Run `docker-compose -f docker-compose-nobuild.yml -f docker-compose.yml up --scale relayer=1 -d`
3. Wait a bit
4. Check `relayer` logs to see error:
```
$ docker-compose logs -f relayer
ops-relayer-1  | node:internal/modules/cjs/loader:936
ops-relayer-1  |   throw err;
ops-relayer-1  |   ^
ops-relayer-1  |
ops-relayer-1  | Error: Cannot find module '/opt/optimism/packages/message-relayer/exec/run-message-relayer.js'
ops-relayer-1  |     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
ops-relayer-1  |     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
ops-relayer-1  |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
ops-relayer-1  |     at node:internal/main/run_main_module:17:47 {
ops-relayer-1  |   code: 'MODULE_NOT_FOUND',
ops-relayer-1  |   requireStack: []
ops-relayer-1  | }
```

**Expected behavior**
The relayer should work

--- a/.changeset/kind-zoos-tell.md
+++ b/.changeset/kind-zoos-tell.md
@@ -1,0 +1,5 @@
+---
+'@eth-optimism/message-relayer': patch
+---
+
+Fixes a bug that prevented the relayer from running correctly.

--- a/ops/scripts/relayer.sh
+++ b/ops/scripts/relayer.sh
@@ -23,4 +23,4 @@ curl \
     $L2_NODE_WEB3_URL
 
 # go
-exec node ./exec/run-message-relayer.js
+exec yarn start

--- a/packages/message-relayer/src/exec/run.ts
+++ b/packages/message-relayer/src/exec/run.ts
@@ -117,4 +117,5 @@ const main = async () => {
 
   await service.start()
 }
-export default main
+
+main()

